<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>possioneditClass</class>
 <widget class="QMainWindow" name="possioneditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>possionedit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>401</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>401</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actiondraw"/>
   <addaction name="actionundo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiondraw">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/new/prefix1/resouses/c5de399d_E864002_8af4615a.png</normaloff>:/new/prefix1/resouses/c5de399d_E864002_8af4615a.png</iconset>
   </property>
   <property name="text">
    <string>draw</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/new/prefix1/resouses/R-C (1).jpg</normaloff>:/new/prefix1/resouses/R-C (1).jpg</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="possionedit.qrc"/>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
